tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/UnitTestRunnerIsTestMethodRunnableIntegration.cs

```diff
@@ -0,0 +1,45 @@
+// <copyright file="UnitTestRunnerIsTestMethodRunnableIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.MsTestV2
+{
+    /// <summary>
+    /// Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.Execution.UnitTestRunner.IsTestMethodRunnable calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = "Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter",
+        TypeName = "Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.Execution.UnitTestRunner",
+        MethodName = "IsTestMethodRunnable",
+        ReturnTypeName = "_",
+        ParameterTypeNames = new string[] { "_", "_", "_" },
```

Comment:
Is it safer to specify these values? e.g. if the next version introduces an overload of `IsTestMethodRunnable` then we might not be able to differentiate between them (as opposed to ignoring the new overload as not required for instrumentation)?

Reply:
You're right, for safety, added in: 27f63fb